scripts/tests.sh

```diff
@@ -1,42 +1,45 @@
+#!/bin/bash
+
 # create tests directory and test case file
 mkdir tests
 touch tests/test_main.py
 
 # add continuity test
-echo '
+cat << EOF > tests/test_main.py
```

Comment:
There's no reason why we should define this script in a string inside of a shell script. This will make modifying and testing this script very annoying. Much easier to have a python file.

Reply:
Pulling from `scripts/test.py` now, deleted `tests.sh`.